src/VisualStudio/Core/Def/Implementation/FindReferences/FindReferencesTableControlEventProcessorProvider.cs

```diff
@@ -42,7 +43,8 @@ public override void PreprocessNavigate(ITableEntryHandle entry, TableEntryNavig
             {
                 if (entry.Identity is ISupportsNavigation supportsNavigation)
                 {
-                    if (supportsNavigation.TryNavigateTo(e.IsPreview))
+                    // TODO: Use a TWD here so we can cancel navigation.
```

Comment:
```suggestion
                    // TODO: Use a Threaded-Wait Dialog here so we can cancel navigation.
```

For sake of anybody else looking at this code.